core-tests/src/test/java/org/mule/runtime/core/message/ErrorBuilderTestCase.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.message;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+import org.mule.runtime.api.message.Error;
+import org.mule.runtime.api.message.ErrorType;
+import org.mule.runtime.api.message.MuleMessage;
+import org.mule.runtime.core.api.DefaultMuleException;
+import org.mule.runtime.core.api.MuleException;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+import org.mule.tck.size.SmallTest;
+
+@SmallTest
+public class ErrorBuilderTestCase extends AbstractMuleTestCase {
+
+  private static final String EXCEPTION_MESSAGE = "message";
+
+  @Test
+  public void buildErrorFromException() {
+    RuntimeException exception = new RuntimeException(EXCEPTION_MESSAGE);
+    Error error = new ErrorBuilder(exception).build();
```

Comment:
for consistency, get the builder with Error.builder() instead of calling the constructor directly